src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * 4;

```

Comment:
The formula for calculating the area of a square is incorrect. The area of a square is the length of a side squared, not multiplied by 4. It should be `return side * side;` or `return Math.pow(side, 2);`.